db/Models/articles-model.js

```diff
@@ -15,5 +15,20 @@ function fetchArticleByID(article_id){
     })
 }
 
+function fetchCommentsByID(article_id){
+    let query = "SELECT * FROM comments WHERE article_id = $1 ORDER BY created_at;"
 
-module.exports = {fetchArticleByID};
\ No newline at end of file
+    return db.query(query, [article_id]).then(({rows}) => {
+        if (rows[0] === undefined) {
+            return Promise.reject({
+                status: 404, 
+                msg : "Article has no comments"
```

Comment:
This error is too generic - 

 - 404 not found - article that doesn't exist 
 - 200 - empty array - article does exist but has no comments 
 
 This is the way that you should be testing this!